Cmdr/CreateGui.lua

```diff
@@ -179,20 +179,33 @@ return function()
 
 	local UIPadding4 = Instance.new("UIPadding")
 	UIPadding4.PaddingLeft = UDim.new(0, 10)
+	UIPadding4.PaddingRight = UDim.new(0, 10)
 	UIPadding4.Parent = TextButton
 
 	local Suggest = Instance.new("TextLabel")
 	Suggest.BackgroundColor3 = Color3.fromRGB(255, 255, 255)
 	Suggest.BackgroundTransparency = 1
 	Suggest.Font = Enum.Font.Code
 	Suggest.Name = "Suggest"
-	Suggest.Size = UDim2.new(1, 0, 1, 0)
+	Suggest.Size = UDim2.new(1, -31, 1, 0)
 	Suggest.Text = "   el"
 	Suggest.TextColor3 = Color3.fromRGB(255, 255, 255)
 	Suggest.TextSize = 14
 	Suggest.TextXAlignment = Enum.TextXAlignment.Left
 	Suggest.Parent = TextButton
 
+	local Footnote = Instance.new("TextLabel")
+	Footnote.BackgroundColor3 = Color3.fromRGB(255, 255, 255)
+	Footnote.BackgroundTransparency = 1
+	Footnote.Font = Enum.Font.Code
+	Footnote.Name = "Footnote"
+	Footnote.Size = UDim2.new(1, 0, 1, 0)
+	Footnote.Text = "Tab→"
```

Comment:
This is bikeshedding so feel free to ignore - but putting a space between Tab and the arrow feels better.

![image](https://github.com/evaera/Cmdr/assets/32818082/64d5fe2b-e014-4c9c-814b-75cd65608890)
![image](https://github.com/evaera/Cmdr/assets/32818082/99d49a5b-4c8d-46b9-a07b-02e28ae72801)

On the other hand, putting a space also feels wrong - just in a different way; I'm torn on this.